osu.Game.Tournament/Components/TournamentBeatmapPanel.cs

```diff
@@ -192,5 +185,86 @@ private void updateState()
                 Alpha = 1;
             }
         }
+
+        private class ModSprite : Container
+        {
+            public string Mod;
+
+            public ModSprite()
+            {
+                Margin = new MarginPadding(10);
+                Width = 60;
+                RelativeSizeAxes = Axes.Y;
```

Comment:
these are specific for the usage in panel, should be inlined instead of set inside component.